### DISCUSSION TITLE
Training a face - best method in this UI?

### DISCUSSION THREAD

**Discussion by telemole** [vladmandic/sdnext#850] [Q&A]
Hi folks, this interface is by far the best - is there a built-in or extension method that produces the best trained face into a model?

Textural inversion? Dreambooth? If anyone can point me to a tutorial that would be amazing - the ones I find are all about 3 months old and show only automatic1111 methods.

I have training images ready but not sure which way to turn now - thanks to this awesome community for all the help!

**Comment by cibernicola** [comment]
IMO LoRA

**Reply by telemole** [reply]
is there anywhere to train it in this interface? can you possibly steer me where to start making a lora? thanks!

**Reply by telemole** [reply]
Working through some tutorials - thanks all who helped out!

**Reply by biship** [reply]
What tutorials?

> Working through some tutorials - thanks all who helped out!
